### PR TITLE
Add needed gems to run Guidelines in development

### PR DESCRIPTION
**Pull request by hcarreras**
Needed to run bundle exec jekyll serve

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -256,16 +256,20 @@ GEM
     unf_ext (0.0.7.7)
     unf_ext (0.0.7.7-x64-mingw32)
     unicode-display_width (1.7.0)
+    webrick (1.8.1)
     zeitwerk (2.6.6)
 
 PLATFORMS
+  arm64-darwin-21
```
Comment (davideluque) [inline Gemfile.lock:263]: In Mac, this can also be ruby

File: Gemfile [hunk 2]
```diff
@@ -5,5 +5,9 @@ group :jekyll_plugins do
   gem 'jekyll-seo-tag'
 end
 
-gem 'wdm', '>= 0.1.0' if Gem.win_platform?
+group :development do
+  gem "jekyll"
+  gem "webrick"
```
Comment (davideluque) [inline Gemfile:10]: Jekyll has a known issue about webrick missing, might have been fixed in newer versions of jekyll

**Comment by MarkusAbtion** [comment]
What would it take for this to get merged?

**Comment by hcarreras** [comment]
Solved conflicts and merged. 
Thanks for the heads up, Markus.

**Pull request merged**